app/components/common/drawer/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { GlobeIcon } from '@radix-ui/react-icons';
+import moment from 'moment';
+import React from 'react';
+import { CalEvent } from '~/utils/interfaces';
+
+interface DrawerProps {
+  event: CalEvent;
+  isDrawerVisible: boolean;
+  toggleDrawer: () => void;
+}
+
+const Drawer: React.FC<DrawerProps> = ({ event, isDrawerVisible, toggleDrawer }) => (
```

Comment:
Nitpick: 
Even if we do not have any logic inside this component, I think it's a good idea to do a regular return instead of this shorthand one to maintain consistency across the components. 

```typescript
const Drawer = (...) => {
    return (
        ...
    )
}
```

Reply:
Yes, fixed it. Actually, I had added return prior but it was give me a eslint error.